tests/pytorch/test_tensorboard_autolog.py

```diff
@@ -0,0 +1,39 @@
+import time
+import mlflow
+import mlflow.pytorch
+
+NUM_EPOCHS = 20
+START_STEP = 3
+
+
+def test_pytorch_autolog_logs_expected_data(tmpdir):
```

Comment:
Do we have a test to check vanilla pytorch autologging is ignored when pytorch-lightining autologging is enabled?

Reply:
Do we have a test to check the version-check warning for pytorch-lightning?

Reply:
i added a check for pytorch autologging being ignored. Checking for warnings proves somewhat tricky: 
* pytest.warns() captures nothing, since mlflow intercepts the warnings and sends them to logging.
* caplog also finds nothing
* capsys/capfd also find nothing, but notably crash pytest.